tests/sass/function-name-format.scss

```diff
@@ -0,0 +1,17 @@
+$foo: kabab-case();
+$foo: snake_case();
+$foo: camelCase();
+$foo: PascalCase();
+$foo: Camel_Snake_Case();
+$foo: SCREAMING_SNAKE_CASE();
+$foo: _with-leading-underscore();
+$foo: _does_NOT-fitSTANDARD();
+
+.class {
+  content: snake_case();
+}
+
+@function camelCase($x, $y) {
```

Comment:
Could you add a declaration like this for each of the cases please as these are the function declarations that we would be expecting to target before the function is actually used. 
It's also good to check that the function names work for all the cases as they are declared as we've seen a few strange parse errors in other rules with certain definitions.